src/common/libutil/unsetenv_glob.c

```diff
@@ -0,0 +1,98 @@
+/************************************************************\
+ * Copyright 2023 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+/* unsetenv_glob.c - unset environment variables matching a glob pattern
+ */
+
+#if HAVE_CONFIG_H
+#include "config.h"
+#endif
+#include <stdlib.h>
+#include <argz.h>
+#include <envz.h>
+#include <fnmatch.h>
+#include <errno.h>
+
+extern char **environ;
+
+static char *split (const char *s, int delim, const char **value)
```

Comment:
`split()` is static to this file, which never calls the function passing a `value`, so that parameter and associated dead code could be dropped.